src/main.rs

```diff
@@ -303,6 +305,11 @@ fn main() {
                             if app_config.print_only {
                                 println!("0x{:x}", rw.desktop_window.x_window_id.unwrap_or(0));
                             } else {
+
+                                for (_, rwd) in &render_windows {
```

Comment:
Please document this section thoroughly and perhaps even link to your issue. Otherwise nobody will have a clue again in two weeks.